### PR TITLE
Refactor ItIntrospectversion class to remove test Order

### PR DESCRIPTION
**Pull request by sankarpn**
Refactor ItIntrospectversion class to remove test order so that individual tests can be run without running the entire test suite.

https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/10181/
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/10180/

### CODE REVIEW

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java [hunk 1]
```diff
@@ -204,215 +227,42 @@ public static void initAll(@Namespaces(2) List<String> namespaces) {
    * under domain status.
    * Verifies that the new pod comes up and sample application deployment works.
    */
-  @Order(1)
   @Test
   @DisplayName("Test introSpectVersion starting a introspector and updating domain status")
   @Tag("gate")
   void testDomainIntrospectVersionNotRolling() {
+    // get the pod creation time stamps
+    LinkedHashMap<String, OffsetDateTime> pods = new LinkedHashMap<>();
```
Comment (rjeberhard) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java:235]: The type of the variable should just be `Map<String, OffsetDateTime>`; don't use the implementation class name here.

**Review by rjeberhard — APPROVED** [review]
Approving... update for the minor comment if you want to.

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java (hunk 1 above)
Comment (robertpatrick) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java:235]: Agreed, please fix this.

**Review by robertpatrick — CHANGES_REQUESTED** [review]
Please address Ryan’s comment

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java (hunk 1 above)
Comment (sankarpn) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java]: Thanks for reviewing, fixed.

**Review by robertpatrick — APPROVED** [review]

**Pull request merged**